routes.js

```diff
@@ -5,6 +5,6 @@ const routes = module.exports = nextRoutes();
 // ========================= APP ROUTES =====================
 routes.add('home', '/', 'index');
 routes.add('panel', '/panel', 'panel');
-routes.add('indicator', '/panel/:indicator', 'panel');
+routes.add('indicator', '/panel/:indicator', 'indicator');
 routes.add('compare', '/compare', 'compare');
```

Comment:
It's not the same design nor the same page. The functionality is different as in one we are showing layers from different indicators in the same map, and the main indicator remains static while in compare I guess each map and widget can change but only shows one at the time (this is a general compare while the first one is a comparison with a specific indicator).
On the other hand, I thought we referred to use the array param to make the call to the api, not in the url. If we can clarify this.
 (Ej: /panel/indicator1?indicators=2,3; Ej: /panel/indicator1) 
 (Ej: /compare?ind1=2,ind2=4,ind3=1; Ej: /compare?ind1=2,ind3=4; Ej: /compare) the 3 maps remind there no matter if are selected or not but at the beginning there isn't data (if url blank).

The call to the API can be de same. (/indicator/:id) or /indicator?filter[id]=1,2,3 if there are more than one at the beginning but if we chose other we should call only the new one to avoid getting and rendering the others that don't change, every time.